pom.xml

```diff
@@ -25,11 +25,12 @@
     </licenses>
 
     <properties>
-        <revision>1.0</revision>
-        <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.277.1</jenkins.version>
-        <java.level>8</java.level>
-        <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
+        <spotbugs.effort>Max</spotbugs.effort>
+		<spotbugs.threshold>Low</spotbugs.threshold>
```

Comment:
Redundant indent

Reply:
>Redundant indent
>
I'll fix it

Reply:
>Redundant indent
>
Done

Reply:
Sorry, it's my mistake. The indent is necessary.

Reply:
There is a conflict on these codes. Could you resolve conflicts? Then, we can merge the PR :)

Reply:
>Sorry, it's my mistake. The indent is necessary.
>
I formatted the code with prettier so i think its alright now 
feel free to suggest change if it's still wrong

Reply:
>There is a conflict on these codes. Could you resolve conflicts? Then, we can merge the PR :)
>
Working on it right now